src/components/IGVBrowser.js

```diff
@@ -38,6 +37,11 @@ const IGVBrowser = ({ selectedFiles, refGenome: { genome, reference }, workspace
     // Requesting a file will store its requester pays status in knownBucketRequesterPaysStatuses.
     const isRequesterPays = await Promise.all(
       _.map(async (url) => {
+        // As seen in requester-pays access URLs resolved from a DRS URI, e.g. AnVIL
+        if (url.startsWith('https') && url.includes('requestedBy=') && url.includes('userProject=')) {
+          return true;
+        }
```

Comment:
```suggestion
        // As seen in requester-pays access URLs resolved from a DRS URI, e.g. AnVIL
        // `userProject` is required to know who to bill
        if (url.startsWith('https') && url.includes('requestedBy=') && url.includes('userProject=')) {
          return true;
        }
```

Reply:
Nice maintainability refinement, done in [7852932](https://github.com/DataBiosphere/terra-ui/pull/5191/commits/7852932915f0079c65f94b0b97eed9beb98f1407).